### ISSUE TITLE
LiteDB Management Tool

### ISSUE THREAD

**Issue by alehofmann** [litedb-org/LiteDB#2016]
Hi!
     I'm looking for a UI tool to manage/admin LiteDB databases.
We've a ongoing complex project where we're leveraging LiteDB NoSql DB, but we need a tool in order to monitor/verify what is actually going on on those database files.

Any help will be greatly appreciated!

**Comment by quiary** [comment]
[litedb studio](https://github.com/mbdavid/LiteDB.Studio), give it a try. it's simple, basic, and the only gui tool I know for litedb management.

**Comment by alemik** [comment]
Hi
This is an old version of my own project: https://alemik.com/
The project is still in progress, but current version is not available. I need many hands to test it :)

**Comment by JKamsker** [comment]
Yes. The official GUI is **LiteDB.Studio**: https://github.com/litedb-org/LiteDB.Studio. You can open a `.db` file, browse collections and documents, and run SQL against it. The main repository also includes a console tool, [LiteDB.Shell](https://github.com/litedb-org/LiteDB/tree/dev/LiteDB.Shell).

The README also lists these community tools:
- OneBella, a cross-platform GUI: https://github.com/namigop/OneBella
- LiteDB.Studio.Avalonia, a C# fork of OneBella: https://github.com/MelnikovIG/LiteDB.Studio.Avalonia

To inspect a database while your application is using it, keep in mind that in the default direct mode the application holds the file open for writing, so Studio can't open it at the same time. Look at it while the app is stopped, or open it with `Connection=Shared` in both your app and Studio (Studio has a "Shared" option in its connection dialog).

Closing this as answered — if it doesn't cover your case, comment here and we'll reopen it.
